.flake8

```diff
@@ -37,6 +37,7 @@ extend-exclude =
 # IMPORTANT: avoid using ignore option, always use extend-ignore instead
 # Completely and unconditionally ignore the following errors:
 extend-ignore =
+  Q  # flake8-quotes makes not sense for black users : https://github.com/zheller/flake8-quotes/issues/105
```

Comment:
Is this needed if flake8-quotes isn't installed?

Reply:
If user install flake8-quotes on system python, vscode will complain about such rules. Adding the "Q" here avoids this issue.